transforms/builder/addAttribute.js

```diff
@@ -0,0 +1,57 @@
+const addAttribute = (path, attribute, value) => (
+  fileInfo,
+  api,
+  options,
+  j,
+  root
+) => {
+  const components = root.findJSXElementsByImport(path);
+
+  if (components.size() === 0) {
+    // If the component is not imported, skip this file
+    return;
+  }
+  if (!components) {
+    return false;
+  }
+
+  let didUpdate = false;
+  components.forEach((component) => {
+    if (
+      !j(component.value.openingElement.attributes)
+        .find(j.JSXIdentifier, { name: attribute })
+        .size()
+    ) {
+      if (typeof value === "string") {
+        component.value.openingElement.attributes.push(
+          j.jsxAttribute(j.jsxIdentifier(attribute), j.literal(value))
+        );
+        didUpdate = true;
+      }
+
+      if (
+        (typeof value !== "string" && typeof value !== "boolean") ||
```

Comment:
Would objects etc not get transformed in this scenario? Might be worth adding some test cases to cover it as well imo

Reply:
I agree, probably cover objects and nodes as well in the tests then we've covered what are probably the most common prop types

Reply:
Yep. Good idea with additional tests. I'll get them written up 👍🏻